Library/Homebrew/requirements/cask_requirement.rb

```diff
@@ -0,0 +1,71 @@
+require "requirement"
+
+$LOAD_PATH.unshift("#{HOMEBREW_LIBRARY_PATH}/cask/lib")
+require "hbc"
```

Comment:
Try to avoid this and require only specific files.

Reply:
This happens because I am working on a kind of outdated fork. I will update them soon, thanks!